elliott/elliottlib/cli/konflux_release_cli.py

```diff
@@ -0,0 +1,93 @@
+import os
+import sys
+from datetime import timedelta
+
+import click
+
+from elliottlib.cli.common import cli, click_coroutine
+from elliottlib.runtime import Runtime
+from doozerlib.constants import KONFLUX_DEFAULT_NAMESPACE
+from doozerlib.backend.konflux_client import KonfluxClient
+from artcommonlib import logutil
+
+LOGGER = logutil.get_logger(__name__)
+
+
+@cli.group("release", short_help="Commands for managing Konflux Releases")
+def konflux_release_cli():
+    pass
+
+
+class WatchReleaseCli:
+    def __init__(self, runtime: Runtime, release: str, konflux_config: dict, timeout: int, dry_run: bool):
+        self.runtime = runtime
+        self.release = release
+        self.konflux_config = konflux_config
+        self.timeout = timeout
+        self.dry_run = dry_run
+        self.konflux_client = KonfluxClient.from_kubeconfig(default_namespace=self.konflux_config['namespace'],
+                                                            config_file=self.konflux_config['kubeconfig'],
+                                                            context=self.konflux_config['context'],
+                                                            dry_run=self.dry_run)
+        self.konflux_client.verify_connection()
+
+    async def run(self):
+        self.runtime.initialize(no_group=True)
+        release_obj = await self.konflux_client.wait_for_release(self.release, overall_timeout_timedelta=timedelta(hours=self.timeout))
+
+        # Assume that these will be available
+        released_condition = next(c for c in release_obj['status']['conditions'] if c['type'] == "Released")
```

Comment:
Need to handle `StopIteration`?

Reply:
I think it is okay to error out. But a better exception can be raised